.github/workflows/stale.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Close stale issues and PRs'
```

Comment:
I'm not sure what to do about this. It seems to be a  change made by @carrodher  in https://github.com/bitnami/charts/commit/3401b1948bf8983c48bf8d47fae725292c847f7f and is getting reflected in this PR for some reason. Maybe he can check and let me know what should be done? I have also checked this file contents against the version in master and they seem identical so I believe it would be merged without issue.